src/main/target/link/stm32_flash_split.ld

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
I see. The first page is only occupied by ISR vector table, larger code occupies bigger flash pages. Smart solution.